scripts/i3-workspace-groups

```diff
@@ -275,6 +284,51 @@ def serve(i3_connection, server_addr):
             connection.close()
 
 
+def get_monitor_active_group(controller, groups_to_workspaces, monitor):
+    active_group = ""
+    min_global = float('inf')
+    for (group, workspaces) in groups_to_workspaces.items():
+        for ws in workspaces:
+            if _get_workspace_field(controller, ws, 'monitor') == monitor:
+                global_number = _get_workspace_field(controller, ws,
+                                                     'global_number')
+                if global_number and global_number < min_global:
```

Comment:
I believe `_get_workspace_field` can return a string; this can cause problems in the `global_number < min_global` comparison. 

![image](https://user-images.githubusercontent.com/69824969/199358689-89d264eb-b99d-43b4-8746-b52215967ec4.png)

Reply:
After looking at the code some more, I believe this is comming from the call to `sorted(parsed_names_dict.keys())`